packages/react/src/ReactNodeViewRenderer.tsx

```diff
@@ -150,6 +150,10 @@ export class ReactNodeView<
       className: `node-${this.node.type.name} ${className}`.trim(),
     })
 
+    if (this.editor.options.addTypeAttributes) {
+      this.renderer.element.setAttribute('data-node-view', '')
+    }
```

Comment:
`props` in ReactRenderer sets attributes to the element, we should use that. This can cause layout shift